source/infrastructure/disaster-recovery/index.html.md.erb

```diff
@@ -191,7 +201,21 @@ GOV.UK Notify is used to communicate with our users via emails, texts and letter
 |Remediation||
 
 ## Google BigQuery
-TBD
+Our services load web analytics and database data into Google BigQuery via an event stream. BQ Data is then available for analysis using various tools.
+
+|||
+|-|-|
+|Impact|Unable to send data to BQ.<br/>Reporting unavailable or out of date.|
+|Prevention||
+|Detection||
```

Comment:
Sentry should alert when there is an error while sending to BQ

Reply:
Contact Google support (how?)

Reply:
added more info